src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
```

Comment:
I have the same problem with static 😢

Reply:
You can declare static field, but you can call it only by className

Reply:
@illiarerun the problem is that if  I will put  'static alive = []' property inside Animals , then eslint will give me this mistake - "error  Parsing error: Unexpected token =" And I won't be able to push my solution.